sample-app/src/main/java/com/joinforage/android/example/ui/pos/screens/ReceiptPreviewScreen.kt

```diff
@@ -0,0 +1,20 @@
+package com.joinforage.android.example.ui.pos.screens
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.viewinterop.AndroidView
+import com.joinforage.android.example.pos.receipts.ReceiptLayout
+import com.joinforage.android.example.ui.pos.ui.ReceiptView
+
+@Composable
+fun ReceiptPreviewScreen() {
+    AndroidView(
+        modifier = Modifier.fillMaxSize(),
+        factory = { context ->
+            ReceiptView(context).apply {
```

Comment:
Kinda wild how well `Compose` plays with `Views + Fragments`. Can't believe you can call `Compose` from within a `Fragment` and then a `Fragment` from within `Compose`!